src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -413,22 +434,105 @@ def save_tile(ix, iy, das):
         # Run in a separate thread
         threading.Thread(target=save_tile, args=(ix, iy, das), ).start()
 
-    def _acquireTile(self, i, ix, iy):
+    def _constructCube(self, images):
+        """
+        Construct xyz cube from a  z stack of images
+        :param images: (list) list of z ordered images
+        :return: (DataArray) the da of the xyz cube
```

Comment:
Please indicate more precisely the types:
* images (list of DataArray of shape YX)
* return (DataArray of shape ZYX)